packages/proto-signing/src/pubkey.ts

```diff
@@ -1,38 +1,37 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { fromBase64 } from "@cosmjs/encoding";
-import { encodeSecp256k1Pubkey, PubKey } from "@cosmjs/launchpad";
+import { encodeSecp256k1Pubkey, PubKey as LaunchpadPubKey } from "@cosmjs/launchpad";
 
-import { cosmos, google } from "./codec";
+import { PubKey } from "./codec/cosmos/crypto/secp256k1/keys";
+import { Any } from "./codec/google/protobuf/any";
```

Comment:
This seems as good as the old version. I know there was a linter rule in tslint which discouraged the use of sub-folder imports. But in other places this is recommended to import a subset of a module.